### ISSUE TITLE
Ectopic gallstone

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10055]
**Preferred term label:**
Ectopic gallstone

**Synonyms**
Gallstone outside of the gallbladder

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Bouveret syndrome (bilioduodenal fistula)
Gallstone ileus

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000455
- label: Ectopic gallstone
- parents: [Cholelithiasis (HP:0001081)](https://hpo.jax.org/app/browse/term/HP:0001081)
- pmid: PMID:27471576
- def: A gallstone located outside of the gallbladder.
- comment: This feature may be seen following laparoscopic cholecystectomy complicated by gallstone and biliary spillage into the peritoneal cavity or with Bouveret syndrome (bilioduodenal fistula).
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10055
- synonyms: Gallstone outside of the gallbladder(NONE)
